Author: Daniel Baumann <mail@daniel-baumann.ch>
Description:
 Updating list of supported filesystems in extlinux manpage (Closes: #692844).

diff -Naurp syslinux.orig/man/extlinux.1 syslinux/man/extlinux.1
--- syslinux.orig/man/extlinux.1
+++ syslinux/man/extlinux.1
@@ -1,17 +1,17 @@
-.TH extlinux "1" "18 December 2007" "SYSLINUX for ext2/ext3 filesystem"
+.TH extlinux "1" "18 December 2007" "SYSLINUX for ext2/ext3/ext4/btrfs filesystem"
 .SH NAME
-extlinux \- install the \s-1SYSLINUX\s+1 bootloader on a ext2/ext3 filesystem
+extlinux \- install the \s-1SYSLINUX\s+1 bootloader on a ext2/ext3/ext4/btrfs filesystem
 .SH SYNOPSIS
 .B extlinux
 [\fIoptions\fP] \fIdirectory\fP
 .SH DESCRIPTION
-\fBEXTLINUX\fP is a new syslinux derivative, which boots from a Linux ext2/ext3
+\fBEXTLINUX\fP is a new syslinux derivative, which boots from a Linux ext2/ext3/ext4 or btrfs
 filesystem.  It works the same way as \fBSYSLINUX\fP, with a few slight modifications.
 It is intended to simplify first-time installation of Linux, and for creation of
 rescue and other special-purpose boot disks.
 .PP
 The installer is designed to be run on a mounted directory.  For example, if you have an
-ext2 or ext3 usb key mounted on /mnt, you can run the following command:
+ext2, ext3, ext4, or btrfs usb key mounted on /mnt, you can run the following command:
 .IP
 .B extlinux --install /mnt
 .SH OPTIONS
